docs/AuthenticationApi.md

```diff
@@ -139,11 +144,14 @@ namespace Example
         {
             Configuration config = new Configuration();
             config.BasePath = "https://api.vrchat.cloud/api/1";
-            var apiInstance = new AuthenticationApi(config);
-            var email = "email_example";  // string | Filter by email. (optional) 
-            var displayName = "displayName_example";  // string | Filter by displayName. (optional) 
-            var username = "username_example";  // string | Filter by Username. (optional) 
-            var excludeUserId = "excludeUserId_example";  // string | Exclude by UserID. (optional) 
+            // create instances of HttpClient, HttpClientHandler to be reused later with different Api classes
```

Comment:
what used to be a simpel 
`var apiInstance = new AuthenticationApi(config);`
now requires 2 more classes
```cs
            HttpClient httpClient = new HttpClient();
            HttpClientHandler httpClientHandler = new HttpClientHandler();
            var apiInstance = new AuthenticationApi(httpClient, config, httpClientHandler);
```
Id recommend you make the HttpClient and HttpClientHandler optional paramters so that existing code will still work out of the box

Reply:
That's not how it works,

You can make a new AuthenticationApi using (ApiClient, ApiClient, Configuration).

HttpClientHandler being exposed to us gives us a great number of control over cookies and proxies.

The wrapper methods take care of all this for you and should be used over manually working with the generated library